pkg/utils/common.go

```diff
@@ -572,3 +573,21 @@ func IsValidSemanticVersion(version string, component string) bool {
 	}
 	return true
 }
+
+func PrintNovelErrors(
```

Comment:
not clear why novel is used in func name

Reply:
added a comment and make a name change to PrintUnreportedErrors